common/libs/contracts/ABIFunction.ts

```diff
@@ -1,6 +1,7 @@
 import abi from 'ethereumjs-abi';
-import { toChecksumAddress, addHexPrefix, stripHexPrefix } from 'ethereumjs-util';
+import { addHexPrefix, stripHexPrefix } from 'ethereumjs-util';
 import BN from 'bn.js';
+import { toChecksumAddress } from 'utils/formatters';
```

Comment:
This could potentially get very confusing considering that the same function name exists in multiple places